client/tests/integration/asset.rs

```diff
@@ -259,8 +259,7 @@ fn find_rate_and_make_exchange_isi_should_succeed() {
 
     let grant_alice_asset_transfer_permission = |asset_id: AssetId, owner_keypair: KeyPair| {
         let allow_alice_to_transfer_asset = GrantBox::new(
-            PermissionToken::new("can_transfer_user_asset".parse().expect("Valid"))
-                .with_params([("asset_id".parse().expect("Valid"), asset_id.clone().into())]),
+            PermissionToken::new("CanTransferUserAsset".to_owned(), &asset_id),
```

Comment:
No more parsing? Don't we have some restrictions for the name?

Reply:
previously it was `Name` so it had to be parsed, now it's `String`

Reply:
Why do you think it shouldn't be `Name` anymore?